members/SWAL22596_666928.yaml

```diff
@@ -0,0 +1,73 @@
+bioguide: SWAL22596_666928
+contact_form:
+  method: post
+  steps:
+    - visit: "https://app.leg.wa.gov/pbc/memberEmail/40/2"
+    - fill_in:
+        - name: street
+          selector: "#Address_Street"
+          value: $ADDRESS_STREET
+          required: true
+        - name: city
+          selector: "#Address_City"
+          value: $ADDRESS_CITY
+          required: true
+        - name: state
+          selector: "#Address_State"
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          required: true
+        - name: zipcode
+          selector: "#Address_Zip"
+          value: $ADDRESS_ZIP5
+          required: true
+    - click_on:
+        - value: Validate address
+          selector: "#ValidateAddressButton"
+    - fill_in:
+        - name: First Name
+          selector: "#FirstName"
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: "#LastName"
+          value: $NAME_LAST
+          required: true
+        - name: emailaddress
+          selector: "#Email"
+          value: $EMAIL
+          required: true
+        - name: confirmemailaddress
+          selector: "#ConfirmEmail"
+          value: $EMAIL
+          required: true
+        - name: phone
+          selector: "#PhoneNumber"
+          value: $PHONE
+          required: false
+        - name: subject
+          selector: "#CommentSubject"
+          value: $SUBJECT
+          required: true
+        - name: message
+          selector: "#Comment"
+          value: $MESSAGE
+          required: true
+    - javascript:
+        - name: phone
+          selectors: [ "#PhoneNumber"]
+          values: [ "$PHONE" ]
+          commands: [ "elements[0].value = values[0].replace(/\\D/g,'').length > 10 ? values[0].replace(/\\D/g,'').substr(1) : values[0].replace(/\\D/g,'')" ]
+          required: false
+    - click_on:
+        - value: Response requested
```

Comment:
I'm assuming we don't care about the secondary responses for this form since we're just leaving it as the default of 'no'?

Reply:
This is selecting the Yes input of "Would you like a response". Not sure what you mean by the "default of no". Maybe I'm missing something?

Reply:
Sorry should have been more clear! After Yes is selected, there is another question that comes up that doesn't look like we care about (which defaults to 'No'). Just wanted to confirm we didn't want to select 'Yes' to that additional q.
![image](https://user-images.githubusercontent.com/19783084/75187449-0a307180-5718-11ea-985e-6603633f40c0.png)

Reply:
oh got it. No, we don't really care about that follow-up question. Good flag tho